### PR TITLE
add(epp): network stormshield api

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-api.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+| Hardware   | Net-Stormshield-Hardware-Api   | Contrôle le matériel                                | X      |            |
+| Health     | Net-Stormshield-Health-Api     | Contrôle l'état de santé                            | X      |            |
+| Interfaces | Net-Stormshield-Interfaces-Api | Contrôle les interfaces                             |        | X          |
+| Uptime     | Net-Stormshield-Uptime-Api     | Contrôle l'uptime                                   | X      |            |
+
+### Règles de découverte
+
+| Nom de la règle                        | Description                                                   |
+|:---------------------------------------|:--------------------------------------------------------------|
+| Net-Stormshield-Api-Interface-Username | Discover network interfaces and monitor bandwidth utilization |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-api.md:30]: description en français?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-api.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+| Métrique    | Unité |
+|:------------|:------|
+| uptime      |       |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Pour la supervision, un utilisateur avec les droits en lecture sur l'API est nécessaire.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-api.md:87]:
```suggestion
Pour la supervision, un utilisateur avec des droits de lecture sur l'API est nécessaire.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-api.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+## Installation
+
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-stormshield-api.md:97]:
```suggestion
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
```

File: pp/integrations/plugin-packs/procedures/network-stormshield-api.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+| Metric Name           | Unit  |
+|:----------------------|:------|
+| system.uptime.seconds | s     |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+To monitor, a user with read privileges on API is required.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-stormshield-api.md:87]:
```suggestion
To monitor, a user with read privileges on the API is required.
```

File: pp/integrations/plugin-packs/procedures/network-stormshield-api.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+the **Configuration > Plugin Packs > Manager** menu.
+
+### Plugin
+
+Since Centreon 22.04, you can benefit from the 'Automatic plugin installation' feature.
+When this feature is enabled, you can skip the installation part below.
+
+You still have to manually install the plugin on the poller(s) when:
+- Automatic plugin installation is turned off
+- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-stormshield-api.md:133]:
```suggestion
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet.
```

File: pp/integrations/plugin-packs/procedures/network-stormshield-api.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+</TabItem>
+</Tabs>
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new host through **Configuration > Hosts**.
+* Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your **Stormshield** server settings.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-stormshield-api.md:168]:
```suggestion
* Fill in the **Name**, **Alias** & **IP Address/DNS** fields according to your **Stormshield** server settings.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**